module/tools/SystemConfiguration/src/SystemConfiguration.cpp

```diff
@@ -181,6 +181,16 @@ namespace module::tools {
                 symlink(link, target);
             }
 
+            /*************
+             * MIRRORLIST *
+             **************/
+            if (config["update_mirrorlist"].as<bool>()) {
+                log<NUClear::INFO>("Updating mirror list to match dockerfile");
+                std::system(
+                    "wget https://raw.githubusercontent.com/NUbots/NUbots/main/docker/etc/pacman.d/mirrorlist \
+                 -O /etc/pacman.d/mirrorlist");
+            }
+
```

Comment:
This is unnecessary, [this file](https://github.com/NUbots/NUbots/blob/69e6f4f12fc5c2468108eeb0ad726f08dca48791/module/tools/SystemConfiguration/CMakeLists.txt) already causes the docker images mirror list to be copied into `SystemConfiguration`s data files, which will automatically result in the mirror list being updated whenever `SystemConfiguration` is run. 

You need to do the extra work in doc/ArchInstall/arch-chroot_install.sh to ensure that `SystemConfiguration` can run in the first place (if you don't sometimes you can't even run `pacman` to downgrade packages to the right versions because there are too many differences in system libraries and package dependencies)